legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/executionPlan/tests/executionPlanTest.pure

```diff
@@ -1472,6 +1472,80 @@ function <<test.Test>> meta::pure::executionPlan::tests::twoDBRenameColumns():Bo
    true;
 }
 
+function <<test.Test>> meta::pure::executionPlan::tests::twoDBTestSpaceQuoting():Boolean[1]
```

Comment:
The PR is changing the Presto extension. But the tests are against H2. 
 
I don't think this test is testing the code change that's been introduced to the Presto extension. The H2 extension is already quoting with double quotes. https://github.com/finos/legend-engine/blob/7dd16e8847cd23f20e31132598632de60bfec6e6/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/h2Extension.pure#L30 

Can we test with Presto ?

Reply:
Can we use TestContainers like we do for SQLServer https://www.testcontainers.org/modules/databases/presto/

Reply:
We have not changed the logic for presto, just refactored the code.

Following two existing tests verify the logic:

https://github.com/finos/legend-engine/blob/61ad833fbf71443e9b8391bde84a388cb0cf3dae/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/transform/fromPure/tests/testToSQLString.pure#L110

https://github.com/finos/legend-engine/blob/61ad833fbf71443e9b8391bde84a388cb0cf3dae/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/transform/fromPure/tests/testToSQLString.pure#L116